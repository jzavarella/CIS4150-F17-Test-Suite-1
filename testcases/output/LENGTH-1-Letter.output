AH0
EY1
===== (1)
DONE! (1 lookups)
