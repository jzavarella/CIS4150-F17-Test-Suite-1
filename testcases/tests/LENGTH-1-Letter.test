#!/bin/sh

./testcases/bin/cmulookup testcases/includes/full-dictionary testcases/includes/external_files/one-letter.txt
