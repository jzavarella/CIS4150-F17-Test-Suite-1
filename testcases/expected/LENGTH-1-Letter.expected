AH0
EY1
=====
DONE! (1 words)
